src/SingleProject/Resizetizer/src/ResizeImageInfo.cs

```diff
@@ -124,6 +128,9 @@ public static ResizeImageInfo Parse(ITaskItem image)
 				info.ForegroundFilename = fgFileInfo.FullName;
 			}
 
+			if (Enum.TryParse<SKFilterQuality>(image.GetMetadata("FilterQuality"), out var filterQuality))
```

Comment:
we might have to add the docs here how to implement in the `MauiImage` and also available values for the `FilterQuality`

Reply:
Done. See https://github.com/dotnet/docs-maui/pull/2604